pkg/statdb/statdb_test.go

```diff
@@ -13,16 +13,14 @@ import (
 	"go.uber.org/zap"
 
 	"storj.io/storj/internal/teststorj"
-	"storj.io/storj/pkg/auth"
 	"storj.io/storj/pkg/statdb"
 	dbx "storj.io/storj/pkg/statdb/dbx"
 	pb "storj.io/storj/pkg/statdb/proto"
 	"storj.io/storj/pkg/storj"
 )
 
 var (
-	apiKey = []byte("")
-	ctx    = auth.WithAPIKey(context.Background(), apiKey)
+	ctx = context.Background()
```

Comment:
nit - it might be better to move this `ctx := context.Background()` into the `TestCreateDoesNotExist` function. It doesn't look like you've run into issues yet but idk I feel like better to make this local.